manifests/gcp_marketplace/chart/kubeflow-pipelines/templates/cache.yaml

```diff
@@ -0,0 +1,235 @@
+apiVersion: apps/v1
+kind: StatefulSet
```

Comment:
any hard reason here it uses StatefulSet? it's the first one start use StatefulSet.

the standalone CLI test: deploy-pipeline-lite.sh line 92 need a check.

Reply:
Previous discussion regarding statefulset: https://github.com/kubeflow/pipelines/pull/3338#pullrequestreview-379975074. The main reason here is because kubeflow is using statefulset for the same job: https://github.com/kubeflow/manifests/blob/master/admission-webhook/bootstrap/base/stateful-set.yaml

Reply:
> deploy-pipeline-lite.sh line 92

that's a good point
please add statefulset to https://github.com/kubeflow/pipelines/blob/a92d52242524309a3e684f46d7f7bb570495362d/test/deploy-pipeline-lite.sh#L92 in the kustomize PR

Reply:
Added in standalone PR.